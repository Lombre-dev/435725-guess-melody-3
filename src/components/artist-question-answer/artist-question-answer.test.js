import React from 'react';
import renderer from 'react-test-renderer';
import ArtistQuestionAnswer from './artist-question-answer';

const INDEX = 0;
const ARTIST = {
  name: `Artist 1`,
  image: ``,
};
const HANDLE_SELECT = () => {};

describe(`<ArtistQuestionAnswer />`, () => {

  it(`render should be match markup`, () => {

    const result = renderer
      .create(<ArtistQuestionAnswer
        index={INDEX}
        artist={ARTIST}
        onSelectCallback={HANDLE_SELECT}
      />)
      .toJSON();

    expect(result).toMatchSnapshot();
  });
});
