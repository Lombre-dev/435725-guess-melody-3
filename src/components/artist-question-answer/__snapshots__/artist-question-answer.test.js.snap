// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ArtistQuestionAnswer /> render should be match markup 1`] = `
<div
  className="artist"
>
  <input
    className="artist__input visually-hidden"
    id="answer-0"
    name="answer"
    onChange={[Function]}
    type="radio"
    value="artist-0"
  />
  <label
    className="artist__name"
    htmlFor="answer-0"
  >
    <img
      alt="Artist 1"
      className="artist__picture"
      src=""
    />
    Artist 1
  </label>
</div>
`;
